src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -209,6 +209,9 @@ export class CalciteCombobox {
   /** Called when the selected items set changes */
   @Event() calciteLookupChange: EventEmitter<HTMLCalciteComboboxItemElement[]>;
 
+  /**
+   * @todo doc
```

Comment:
Who should add this in?

Reply:
IMO this could be an internal event. It's a weird feature to let the dev know when they changed the markup as they theoretically _just performed_ the dom manipulation themselves. It's really just for registering the items...